Tilers/CityTiler/citym_cityobject.py

```diff
@@ -241,6 +303,24 @@ def retrieve_geometries(cursor, city_object_ids, offset, objects_type):
             arrays.append({
                 'position': geom.getPositionArray(),
                 'normal': geom.getNormalArray(),
-                'bbox': [[float(i) for i in j] for j in geom.getBbox()]
+                'bbox': [[float(i) for i in j] for j in geom.getBbox()],
+                'uv': geom.getDataArray(0)
             })
-        return arrays
+        return arrays, key
+
+def calculationSurface(size):
```

Comment:
calculationSurface, imageConvertToPng and multipleOf2 seem to be helper functions
of retrieve_geometries(). Place them in another file. Maybe in Altas.py that is already
imported ?

Reply:
done in 6a5720e9dc6ce7cbc139642aebd588fb47eb88ff